packages/components/src/components/post-header/post-header.tsx

```diff
@@ -307,15 +305,22 @@ export class PostHeader {
 
   private renderNavigation() {
     const navigationClasses = ['navigation'];
+
+    const mobileMenuScrollTop = this.mobileMenu?.scrollTop ?? 0;
+
     if (this.mobileMenuExtended) {
       navigationClasses.push('extended');
     }
-    if (!this.megadropdownOpen) {
-      navigationClasses.push('scroll-y');
+
+    if (this.megadropdownOpen) {
+      navigationClasses.push('megadropdown-open');
     }
 
     return (
-      <div class={navigationClasses.join(' ')}>
+      <div
+        class={navigationClasses.join(' ')}
+        style={{ '--header-navigation-scroll-top': `${mobileMenuScrollTop}px` }}
```

Comment:
This will be updated only when the `renderNavigation` is called so it would have an incorrect value if you just scroll the page.

Reply:
This is intentionally, because the custom-property is only needed, if the mobile menu is open, the user scrolled down the main navigation level and then opens the second navigation level.

Reply:
Ok! This could be renamed to something like `--header-navigation-initial-inset`, just to make it clear that it the value is ot dynamic in opposition to `--header-scroll-top` that is.